api/v1alpha1/lvmcluster_types.go

```diff
@@ -108,6 +114,11 @@ type LVMClusterStatus struct {
 	// Ready describes if the LVMCluster is ready.
 	// +optional
 	Ready bool `json:"ready,omitempty"`
+
+	// Reason describes the reason behind any success or failure.
+	// +optional
+	Reason string `json:"reason,omitempty"`
```

Comment:
Do we add `reason` for error specific to this PR only?
If that's the case, then we need to add reasons for other failures as well in a separate PR.

Reply:
I am updating the reason wherever required.